pubsub/google/cloud/pubsub/connection.py

```diff
@@ -539,3 +543,22 @@ def test_iam_permissions(self, target_path, permissions):
         path = '/%s:testIamPermissions' % (target_path,)
         resp = conn.api_request(method='POST', path=path, data=wrapped)
         return resp.get('permissions', [])
+
+
+def _transform_messages_base64(messages, transform, key=None):
+    """Helper for base64 encoding and decoding messages.
+
+    :type messages: list
+    :param messages: List of dictionaries with message data.
+
+    :type transform: :func:
```

Comment:
Sphinx won't build docs for a non-public function, so this doesn't have to be valid Sphinx but I don't think `:func:` would build.

I think that :class:`~types.FunctionType` would work, and maybe Sphinx is even cool with `function` (though `function` is not a Python builtin).